README.md

```diff
@@ -52,8 +52,7 @@ To install CloudGoat, make sure your system meets the requirements above, and th
 ```
 $ git clone https://github.com/RhinoSecurityLabs/cloudgoat.git
 $ cd cloudgoat
-$ pip3 install -r ./core/python/requirements.txt
-$ chmod u+x cloudgoat.py
```

Comment:
chmod isn't needed anymore as ./cloudgoat.py has the executable bit set in the repo now.